synapse/pipeline.yml

```diff
@@ -183,7 +189,7 @@ steps:
         - exit_status: 2
           limit: 2
 
-  - label: ":python: 3.7 / :postgres: 9.5"
+  - label: ":python: 3.7 / :postgres: 11"
```

Comment:
Why this change? Why are we testing pg11?

Reply:
Github isn't doing the best job of showing a sensible diff.

I was thinking it would be sensible to test against a range of postgres versions rather than just min and max, hence I'm suggesting we retain the current tests against py3.7/pg11.

Rather, I'm replacing the tests against  py3.7/pg9.5 with tests against py3.8/pg12; the rationale being that we already testing against pg9.5 (in the py3.5/pg9.5 tests), and I'm not super-keen to introduce even more test runs.

Reply:
Ooooh, right yes. Agreed.